src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -35,7 +35,7 @@ public static ArgumentMultimap tokenize(String argsString, Prefix... prefixes) {
      * @param prefixes   Prefixes to find in the arguments string
      * @return           List of zero-based prefix positions in the given arguments string
      */
-    private static List<PrefixPosition> findAllPrefixPositions(String argsString, Prefix... prefixes) {
+    public static List<PrefixPosition> findAllPrefixPositions(String argsString, Prefix... prefixes) {
```

Comment:
Currently, the Argument Multimap stores prefixes unorderedly, so I can't retrieve the sequence of sort keys that I want. So I made this public so that the Sort Command Parser can use it.